src/views/HomePage.tsx

```diff
@@ -42,20 +46,70 @@ export default class Home extends React.Component<HomeProperties> {
   //   this.props.mousePositionStore.setPosition({ x: e.clientX, y: e.clientY });
   // };
 
+  handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    this.props.appModel.textInput = e.target.value;
+  }
+
   render() {
+
+    //TODO: delete after reading.
+    // we can extract the appModel to avoid boilerplate :  this.props.appModel.<some_prop>
+    const { appModel } = this.props;
+    // We can even extract some re-usable props from appModel to follow DRY
+    const { textInput, selectedItem } = appModel;
+
     //const { comboboxStore } = this.props;
 
     //const { query, items, selectedItem } = comboboxStore;
     // MousePositionView
 
     return (
-        <div className="home">
-          <h2>Home Page</h2>
-          <h3>Mouse Position: 
-            {this.props.appModel.mousePosition.x}, 
-            {this.props.appModel.mousePosition.y}</h3>
-        </div>
-    );
+      <div className="home">
+        <h2>Home Page</h2>
+        <h3>Mouse Position: 
+          {this.props.appModel.mousePosition.x}, 
+          {this.props.appModel.mousePosition.y}</h3>
+
+        <br />
```

Comment:
Maybe just an <hr> here to make the code have fewer lines

Reply:
done